apollo-compiler/src/test/graphql/com/example/input_object_type/TestQueryExpected.java

```diff
@@ -0,0 +1,86 @@
+package com.example.input_object_type;
+
+import com.apollostack.api.Query;
+import java.lang.Override;
+import java.lang.String;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+public final class TestQuery implements Query<TestQuery.Variables> {
```

Comment:
That's what we do on iOS as well, `GraphQLQuery` and `GraphQLMutation` extend `GraphQLOperation`.